packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -395,6 +397,7 @@ class DeactivateAccountReason extends React.Component {
                                         placeholder={localize('What could we do to improve?')}
                                         name='doToImprove'
```

Comment:
Can you change this to `snake_case`? It's not part of your change but we should follow the naming conventions defined [here](https://github.com/binary-com/deriv-app/tree/master/docs/JavaScript#readme)

Reply:
I've updated the other one as well. 

      otherTradingPlatforms = `other_trading_platforms`
      doToImprove = `do_to_improve`